teamC/dayoung/app.js

```diff
@@ -31,7 +31,120 @@ appDataSource.initialize()
 
 app.get('/ping', function(req, res, next){
   res.json({message : 'pong'})
-})
+});
+
+
+app.get('/lookup', async(req, res) =>{
+    await appDataSource.manager.query(
+      `SELECT 
+        users.id, 
+        users.profile_image, 
+        posts.id, posts.user_id, 
+        posts.title, posts.content 
+      FROM users, posts
+      WHERE users.id = posts.user_id`
+      ,(err, rows) => {
+        res.status(200).json({"data" : rows});
+      }
+    )
+  });
+
+  app.get('/userInfo/:user_id', async(req, res) =>{
+    const userId = req.params.user_id;
+    const result = {};
+    const postingList = [];
+    await appDataSource.manager.query(`
+        SELECT 
+            users.id as userId, 
+            users.profile_image as userProfileImage, 
+            posts.id as postingId, 
+            posts.title, 
+            posts.content 
+        FROM users inner join posts on users.id = posts.user_id and users.id = ${userId}`
+      ,(err, rows) => {
+         result.usersId = rows[0].userId;
+         result.useProfileImage = rows[0].userProfileImage;
+         rows.map(el =>{
+          let tmp = {};
+          tmp.postingId = el.postingId;
+          tmp.content = el.content;
+          tmp.title = el.title;
+          postingList.push(tmp);
+         })
+         result.postings = postingList;
+        res.status(200).json({"data" : result});
+      }
+    )
+  });
+  
+app.post("/users", async(req, res, next) => {
+    const {name, email, profile_image, password} = req.body;
+    await appDataSource.query(`INSERT INTO users(
+        name, 
+        email, 
+        profile_image, 
+        password
+        )values(?, ?, ?, ?);`,
+      [name, email, profile_image, password]
+    );
+    res.status(201).json({"message" : "userCreated"});
+  });
+
+  app.post("/posts", async(req, res, next) => {
+    const {title, content, user_id} = req.body;
+    await appDataSource.query(`
+      INSERT INTO posts(
+        title, 
+        content, 
+        user_id
+        )values(?, ?, ?);`,
+      [title, content, user_id]
+    );
+    res.status(201).json({"message" : "postCreated"});
+  });
+
+  app.post("/likes/:user_id", async(req, res, next) => {
+    userId = req.params.user_id;
+    const {post_id} = req.body;
+    await appDataSource.query(`
+      INSERT INTO likes(
+        user_id, 
+        post_id
+        )values(${userId}, ?);`,
+      [post_id]
+    );
+    res.status(201).json({"message" : "likeCreated"});
+  });
+
+  app.patch("/editPost/:post_id", async(req, res, next) => { // next의 역할은 무엇인가?
+    const postId = req.params.post_id;
```

Comment:
javascript에서는 snake_case가 아니라 camelCase를 사용해주세요~
```suggestion
    const postId = req.params.postId;
```

Reply:
수정했습니다!